### DISCUSSION TITLE
Docs 'Attaching files' has missing important note

### DISCUSSION THREAD

**Discussion by mbiesiad** [community/community#31346] [Other Feature Feedback, Questions, & Ideas]
Article on docs.github.com:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

At this moment there is a `Warning` with info, but there is not mentioned about that - even if user will Cancel own changes, will not create an issue and so on so on, but just drag&drop a file during issue's creation process, it will be still uploaded. It looks quite dangerous (taking into account possible mistakes).

Example:
https://user-images.githubusercontent.com/18367606/187424575-d4a4e50d-7c57-40d6-ae40-1a29ba0868dd.png

The simplest scenario:

User are creating an issue, and using drag&drop option added 'cat.jpg', user realised that it was INCORRECT file, so deleted a lines and add a 'dog.jpg'. Result - both of the files are uploaded.

The user should be **warned** about that.

Additional information
It's a little bit similar with: https://github.com/orgs/community/discussions/28316

Typical two ways for this:
- first one - if I drag & drop (so upload file process) I can edit or delete/remove this (mistake case) later
- second one - if I wanted to upload something and I will not to have an opportunity to delete it in easy way, I can see a confirmation window ("Are you sure?")

Here it's focus on missing note about this and possible solutions other than tickets for Support.

Best wishes,

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 

You're able to edit docs yourself, if you'd like to create a pull request to suggest these changes, use the Make a contribution button at the bottom of the page! 

Thanks for helping improve our docs 😄 :octocat:
